azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/util/BlobHelper.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.arm.util;
+
+import org.jclouds.ContextBuilder;
+import org.jclouds.azure.storage.domain.BoundedSet;
+import org.jclouds.azureblob.AzureBlobClient;
+import org.jclouds.azureblob.domain.BlobProperties;
+import org.jclouds.azureblob.domain.ContainerProperties;
+import org.jclouds.azureblob.domain.ListBlobsResponse;
+import org.jclouds.azurecompute.arm.domain.VMImage;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BlobHelper {
+
+   public static void deleteContainerIfExists(String storage, String key, String containerName) {
+      final AzureBlobClient azureBlob = ContextBuilder.newBuilder("azureblob")
+              .credentials(storage, key)
+              .buildApi(AzureBlobClient.class);
+
+      azureBlob.deleteContainer(containerName);
+
+      try {
+         azureBlob.close();
+      } catch (IOException e) {
+         e.printStackTrace();
+      }
+   }
+
+   public static boolean customImageExists(String storage, String key) {
+      boolean retval = false;
+      final AzureBlobClient azureBlob = ContextBuilder.newBuilder("azureblob")
+              .credentials(storage, key)
+              .buildApi(AzureBlobClient.class);
+
+      if (azureBlob.containerExists("system"))
```

Comment:
Just ` return azureBlob.containerExists("system")`; also wrap it in a try/finally block to close the context.